docs/SRS/SRS.bak0

```diff
@@ -0,0 +1,517 @@
+\documentclass[12pt]{article}
```

Comment:
idk why this was created with the `.bak0` extension. It looks like your `latexindent` command was not picking up the `.latexindent.yaml`, since that should have it create this file under `.bak` which will then match the `.gitignore`. Either way, remember to delete this file.